src/test/spec/transactions.rs

```diff
@@ -27,66 +26,6 @@ async fn run_unified() {
     run_spec_test(&["transactions", "unified"], run_unified_format_test).await;
 }
 
-#[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-#[function_name::named]
-/// This test replicates the test cases in errors-client.json as we do not have the document
-/// validation required to trigger client-side errors in those tests.
```

Comment:
This issue is no longer relevant as of [this commit](https://github.com/mongodb/specifications/commit/a124e21be89cbfc78606725c6ed1fa00dc992777); we currently have `errors-client.json` synced and running, so we don't need this additional test.